vendor/pyne_source_sampling/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Create a library called "pyne_source_sampling".
+
+# Find DAGMC/MOAB
+find_package(DAGMC REQUIRED)
```

Comment:
Since this is already called in the root-level CMakeLists.txt, the variables there should be available when this file is run. i.e., I think you can remove this and it will still work.